lib/extension.js

```diff
@@ -73,4 +92,19 @@ function isAllowedInParent(property, parent) {
 
 function isWildcard(allowedIn) {
   return allowedIn.indexOf(WILDCARD) !== -1;
+}
+
+function isMessageEvent(event) {
+  const eventDefinitions = event.get('eventDefinitions');
+
+  return eventDefinitions.some(function(def) {
+    return is(def, 'bpmn:MessageEventDefinition');
+  });
+}
+
+function isZeebeServiceTaskSpecific(property) {
```

Comment:
Minor, but in order to make this more understandable, I think the function actually should be called `isAllowedInZeebeServiceTask` (because a property might still be allowed in other taskTypes as well, so `specific` is not the right term here)

Reply:
I agree that maybe the name is not the best, but this is related to the question regarding the task array. It's the ones that are allowed for ZeebeServiceTask but that are not allowed already for all events.

Reply:
Proposal: Change to the name suggested by Max and document in the function why mere `is(element, "zeebe:ZeebeServiceTask")` check is not enough.